sdk/src/wallet/operations/output_claiming.rs

```diff
@@ -244,7 +243,10 @@ where
             .iter()
             .map(|i| i.output.amount())
             .sum::<u64>()
-            >= MinimumStorageDepositBasicOutput::new(rent_structure, token_supply).finish()?;
+            >= BasicOutputBuilder::new_with_minimum_amount(storage_score_params)
+                .add_unlock_condition(AddressUnlockCondition::new(Ed25519Address::null()))
+                .finish()?
+                .amount();
```

Comment:
Maybe would be nice to have a shorter way for this

Reply:
Not possible anymore

Reply:
And another